styles.css

```diff
@@ -153,12 +154,20 @@ ul {
   align-self: flex-end;
 }
 
+.my-recent-work {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 375px;
```

Comment:
Kindly, I suggest you not to use fixed width, it is better to use percentages so that your site adapts to all types of mobile devices.